src/cljs/aikakonematka/components.cljs

```diff
@@ -56,50 +58,51 @@
-          :on-click #(do
-                       (rf/dispatch [:set-game-img "images/puzzleImage.jpg"])
-                       (util/show-game!))}]
-   (into [:ul
-          {:style {:position "absolute"
-                   :z-index  "1"}}
-          [:li [:a
-                {:href     "#!"
-                 :on-click #(do
-                              (rf/dispatch [:set-game-img "images/puzzleImage.jpg"])
-                              (util/show-game!))}
-                "default"]]]
-         (map (fn [search-word]
-                ^{:key search-word} [:li [:a {:href     "#"
-                                              :on-click #(do
-                                                           (set-game-image! search-word) (util/show-game!))}
-                                          search-word]])
-              ["kirkko"
-               "miehet"
-               "naiset"
-               "sotilas"
-               "rauta"]))])
+  (into
+    [:div
+     [:img {:style  {:position "absolute"
+                     :z-index  "0"
+                     :display  "block"}
+            :src    "images/puzzle-selection-bg.png"
+            :width  "100%"
+            :height "100%"}]
+     [:img {:style    {:position "absolute"
+                       :z-index  "1"
+                       :left     "18.5%"
+                       :top      "13.7%"}
+            :src      "images/puzzleImage.jpg"
+            :width    "20%"
+            :height   "27.5%"
+            :href     "#!"
+            :on-click #(do
+                         (rf/dispatch [:set-game-img ["default" "images/puzzleImage.jpg"]])
+                         (util/show-game! "default"))}]]
+    (map (fn [{:keys [search-word left top]}]
+           (set-game-image! search-word)
+           ^{:key search-word} [:img
+                                {:id       search-word
+                                 :style    {:position "absolute"
+                                            :z-index  "1"
+                                            :left     left
+                                            :top      top}
+                                 :src      (let [game-imgs @(rf/subscribe [:game-img])]
+                                             (when game-imgs
+                                               (game-imgs search-word "")))
+                                 :width    "20%"
+                                 :height   "27.5%"
+                                 :on-click #(util/show-game! search-word)}])
+         [{:search-word "kirkko" :left "39%" :top "13.7%"}
+          {:search-word "miehet" :left "60%" :top "13.7%"}
+          {:search-word "naiset" :left "18.5%" :top "43.5%"}
+          {:search-word "sotilas" :left "39%" :top "43.5%"}
+          {:search-word "rauta" :left "60%" :top "43.5%"}])))
 
 (defn app []
   (if (and (= :game @(rf/subscribe [:screen]))
-           (string? @(rf/subscribe [:game-img])))
+           (string? @(rf/subscribe [:game-play-image])))
```

Comment:
(@(rf/subscribe [:game-img]) @(rf/subscribe [:game-play-image])) needs to be "string" ?:)